test/lsi/lsi_test.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 require File.dirname(__FILE__) + '/../test_helper'
+# require_relative '../test_helper'
+# require 'debug'
```

Comment:
This looks like it might be debugging code left over. Would you mind removing it?